doc/guides/technical-priorities.md

```diff
@@ -0,0 +1,121 @@
+# Technical Priorities
+
+This list represents the current view of key technical priorities recognized
+by the project as important to ensure the ongoing success of Node.js.
+It is based on an understanding of the Node.js
+[constituencies](https://github.com/nodejs/next-10/blob/main/CONSTITUENCIES.md)
+and their [needs](https://github.com/nodejs/next-10/blob/main/CONSTITUENCY-NEEDS.md).
+
+The initial version was created based on the work of the
+[Next-10 team](https://github.com/nodejs/next-10) and the
+[mini-summit](https://github.com/nodejs/next-10/issues/76)
+on August 5th 2021.
+
+They will be updated on a regular basis and will be reviewed by the next-10 team
+and the TSC on a 6 month basis.
+
+* Modern HTTP
```

Comment:
I would recommend to use `##` instead of `*` as the former automatically create an anchor in github.